### ISSUE TITLE
Suggestion - gun obtainability/repairs rework

### ISSUE THREAD

**Issue by eatAnus** [InZidiuZ/op-framework-issue-tracker#3174]
### Assurance

I have read the `README.md`

### Summary

Guns are too accessible and last too long making them lose value in multiple ways, seeing a AR doesn't feel special anymore, with the way things are new people can join the server and just buy a insane gun with their starting balance or practically get it free from daily/spin. Please consider redoing how all guns are obtained and upkept, guns shouldn't be dropped for free from dailies/spins and be so cheap with how the economy is, it's just doesn't feel right, the amount of guns that are just thrown around without any care.

___________________________________________________________________________________________________________________________________________
**Guns rework ideas**

- Remove guns from gun shop and require a firearm license, also maybe look in to prices
![image](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/46156400/9e93906b-8a6b-49f5-ad33-fd66dcacf00f)
[compare to PD prices]

- Instead of finding fully functional guns in trash or other misc. activities, players would instead stumble upon broken/worn guns

- Remove guns from daily rewards and spins would make obtaining them a more deliberate choice and not just a matter of luck or convenience. This would prevent players from accumulating an excess of weapons without any effort or investment

- ~~Remove PD/EMS restrictions on guns/items, shouldn't have to worry about people taking things we have our lovely staff to step in if necessary, PD prices should also be updated to be more realistic~~
![image](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/46156400/10e351c1-232d-4075-84f9-87b86a666411)
[compare to gunstore prices]
___________________________________________________________________________________________________________________________________________

**Gun decay/repair**

- Allow for guns to be repaired with a bench/item for longevity of a weapon similar to Rust (also maybe have another bench/item that would get rid of the drawback of ware and would then fully repair the gun. Also could have it so that one or the other is permanent and after so many repairs your gun is fully broken)
![image](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/46156400/e118d0d2-0222-4da4-8987-0186ffd1c08f)
[the drawback would be similar to this and would add either decay or ware]

- Make gun decay/ware a separate system when guns are being shot the guns will slightly gain ware and when you reload it will gain more ware. Decay could happen whenever it sits in storage
![image](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/46156400/efe2b457-6a59-440e-876d-72017981c7b1)
![image](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/46156400/2939ab49-a092-4697-a8f6-f84b4655b047)
![image](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/46156400/b9a8976b-b14f-46a9-9c61-3068edb02210)
![image](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/46156400/702e119c-d74c-49c1-9ae8-313090426a15)
[these aren't in any sort of order, just examples of ware and decay]
[decay = red / ware = purple]

- ~~make ALL current existing guns have random ware/decay~~

### Reason

Might seem troubling for some people but I could see a gun rework adding so much to RP for civs, crims and cops, introducing a increased demand for guns can actually enhance roleplay experiences in so many ways. People will be encouraged to explore alternative conflict methods and diversifying interactions (not being able to just go around shooting for no reason as it will effect their gun.) These concepts can also spur unique storylines and character arcs, fostering immersion and personal growth along with new satisfying opportunities. We must prioritize everyone's RP experience and not anyone in particulars certain experience. This would also effect the economy in a great way as people have way too much money, everyone is rich and too worried about export vehicles.. let's try and fix the state of guns so that it has that special feeling again, please consider.

_TLDR;
nerf gun obtainability, more interactions, better roleplay_

**Comment by eatAnus** [comment]
![image](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/46156400/dee284bb-2b74-4f33-9d9c-44d8ad8688d3)
1 day of obtaining guns example

**Comment by coalaura** [comment]
> Remove guns from gun shop and require a firearm license, also maybe look in to prices

We already have a few better legal guns that are locked behind a license. I don't like the idea of making all guns require a license.

> Instead of finding fully functional guns in trash or other misc. activities, players would instead stumble upon broken/worn guns

I'll make guns found in the wild be already slightly degraded.

> Remove PD/EMS restrictions on guns/items, shouldn't have to worry about people taking things we have our lovely staff to step in if necessary, PD prices should also be updated to be more realistic

You already striked it yourself but again, no.

> Allow for guns to be repaired with a bench/item for longevity of a weapon similar to Rust (also maybe have another bench/item that would get rid of the drawback of ware and would then fully repair the gun. Also could have it so that one or the other is permanent and after so many repairs your gun is fully broken)

No, already been discussed many times before.

> Make gun decay/ware a separate system when guns are being shot the guns will slightly gain ware and when you reload it will gain more ware. Decay could happen whenever it sits in storage

same as above, no

**Comment by coalaura** [comment]
dailies is its own thing so regarding removing guns from its drops i'd make a seperate issue itself

**Comment by eatAnus** [comment]
feel like this could of had more feedback/discussion from others as this wasnt up long

**Comment by coalaura** [comment]
> feel like this could of had more feedback/discussion from others as this wasnt up long

thats cause this exact topic has already been discussed multiple times previously
